prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
```

Comment:
In general, we want to avoid styling all div elements like this. Instead you can target the div with the id attribute of card-deck like this

```css
#card-deck {

}
```

Check your other files too.

https://github.com/andrewthoang/prep-0922-code-solutions/pull/8/files